.changeset/friendly-pens-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen-codegen': minor
```

Comment:
`patch`?

Reply:
It's OK to have `minor` in hydrogen-codegen. Since this is a new feature, it should be minor I think?